spec/features/admin/appropriate_bodies/users_spec.rb

```diff
@@ -109,11 +134,19 @@ def and_i_see(string)
     expect(page).to have_content(string)
   end
 
+  def then_i_should_not_see(string)
+    expect(page).not_to have_content(string)
+  end
+
   def and_i_should_not_see(string)
     expect(page).not_to have_content(string)
   end
 
   def given_a_appropriate_body_user_exists
     @appropriate_body_user = create(:user, :appropriate_body)
   end
+
+  def when_i_fill_in(selector, with:)
```

Comment:
One for another PR but we seem to duplicate a lot of these helper methods; we could DRY it all up if we extract them